src/gnssro/gnss_tec_ground_based.py

```diff
@@ -0,0 +1,469 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+#
+# decode TENET formatted ground-based GNSS Total Electron Content (TEC)
+#
+
+import sys
+import os
+import time
+from datetime import datetime, timezone
+import numpy as np
+import logging
+
+# These modules need the path to lib-python modules
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.orddicts import DefaultOrderedDict
+from pyiodaconv.def_jedi_utils import iso8601_string, epoch
+
+os.environ["TZ"] = "UTC"
+
+# these are the unique values in the raw input file
+varDict = {'totalElectronContentVertical': ['totalElectronContentVertical', "integer", 'TECU'],
+           'totalElectronContentSlant': ['totalElectronContentSlant', "integer", "TECU"],
```

Comment:
@davidkuhl do we ignore the `Vertical` TEC, we only care about the `Slant` TEC correct?

Reply:
@BenjaminRuston Yes, we are only interested in assimilating the Slant TEC and verifying against the Slant TEC.